### ISSUE TITLE
`filezilla` needs `fzfstp`

### ISSUE THREAD

**Issue by beroal** [roddhjav/apparmor.d#618]
There is no `fzfstp` profile. I believe it existed a few years ago since I used FileZilla before. I botched a profile for myself and can send a pull request.

**Comment by roddhjav** [comment]
Good catch, I try to regularly remove old/not maintained profiles. If you don't mind, Filezilla is probably the next that will be removed.
